src/main/java/com/netuitive/dropwizard/metrics/NetuitiveMetricsCloudReporter.java

```diff
@@ -0,0 +1,323 @@
+         * @return {@code this}
+         */
+        public Builder convertRatesTo(final TimeUnit _rateUnit) {
+            this.rateUnit = _rateUnit;
+            return this;
+        }
+
+        /**
+         * Convert durations to the given time unit.
+         *
+         * @param _durationUnit a unit of time
+         * @return {@code this}
+         */
+        public Builder convertDurationsTo(final TimeUnit _durationUnit) {
+            this.durationUnit = _durationUnit;
+            return this;
+        }
+
+        /**
+         * Only report metrics which match the given filter.
+         *
+         * @param _filter a {@link MetricFilter}
+         * @return {@code this}
+         */
+        public Builder filter(final MetricFilter _filter) {
+            this.filter = _filter;
+            return this;
+        }
+
+        /**
+         * Builds a {@link NetuitiveMetricsCloudReporter} with the given properties, sending metrics to Netuitive Cloud using the API key.
+         *
+         * @param apiKey the API key of the Netuitive Cloud Datasource.
+         * @param apiHost the host name of the Netuitive Cloud API endpoint. (optional)
+         * @param elementName the element name to be updated
+         * @param elementType the element type of the element. (optional)
+         * 
+         * @return a {@link NetuitiveMetricsCloudReporter}
+         */
+        public NetuitiveMetricsCloudReporter build(final String apiKey, final String apiHost, final String elementName, final String elementType) {
+            return new NetuitiveMetricsCloudReporter(registry, 
+                                                apiKey, 
+                                                apiHost, 
+                                                elementName,
+                                                elementType,
+                                                this.rateUnit, 
+                                                this.durationUnit, 
+                                                this.filter);
+        }
+
+    }
+
+    /**
+     * Called periodically by the polling thread. Subclasses should report all the given metrics.
+     *
+     * @param gauges     all of the gauges in the registry
+     * @param counters   all of the counters in the registry
+     * @param histograms all of the histograms in the registry
+     * @param meters     all of the meters in the registry
+     * @param timers     all of the timers in the registry
+     */
+    @SuppressWarnings("rawtypes")
+    @Override
+    public void report(SortedMap<String, Gauge> gauges,
+                       SortedMap<String, Counter> counters,
+                       SortedMap<String, Histogram> histograms,
+                       SortedMap<String, Meter> meters,
+                       SortedMap<String, Timer> timers) {
+        
+        
+        LOG.info("Pushing {} gauges, {} counters, {} histograms, {} meters and {} timers to Netuitive",
```

Comment:
Is there an option to turn off logging? Logging from a library can sometimes be annoying.